apps/dashboard/src/pages/api-keys.tsx

```diff
@@ -36,80 +40,149 @@ export function ApiKeysPage() {
-                      <div className="flex items-center gap-2">
-                        <InputField className="flex overflow-hidden pr-0">
-                          <Input
-                            className="cursor-default"
-                            value={showApiKey ? form.getValues('apiKey') : maskApiKey(form.getValues('apiKey'))}
-                            readOnly
-                          />
-                          <CopyButton size="input-right" valueToCopy={form.getValues('apiKey')} />
-                        </InputField>
-
-                        <Button
-                          variant="outline"
-                          size="icon"
-                          onClick={toggleApiKeyVisibility}
-                          aria-label={showApiKey ? 'Hide API Key' : 'Show API Key'}
-                        >
-                          {showApiKey ? <RiEyeOffLine className="size-4" /> : <RiEyeLine className="size-4" />}
-                        </Button>
-                      </div>
-                      <p className="text-foreground-600 text-xs">
-                        Use this key to authenticate your API requests. Keep it secure and never share it publicly.
-                      </p>
-                    </div>
-
-                    <div className="space-y-2">
-                      <label className="text-foreground-600 text-sm font-medium">Application Identifier</label>
-                      <div className="flex items-center gap-2">
-                        <InputField className="flex overflow-hidden pr-0">
-                          <Input className="cursor-default" value={form.getValues('identifier')} readOnly />
-                          <CopyButton size="input-right" valueToCopy={form.getValues('identifier')} />
-                        </InputField>
-                      </div>
-                      <p className="text-foreground-600 text-xs">
-                        The public application identifier used for the Inbox component
-                      </p>
-                    </div>
-                  </div>
-                </CardContent>
-              </Card>
-            </Form>
-            <div className="column flex gap-2 p-6 pt-0">
+        <Container>
+          <div className="grid grid-cols-1 gap-6 lg:grid-cols-[224px,1fr]">
+            <div className="column flex gap-2 pt-0">
               <div className="flex flex-col gap-2">
-                <RiKey2Line className="h-10 w-10" />
-                <h2 className="text-foreground-950 text-lg font-medium">Environment Keys</h2>
-                <p className="text-foreground-400 text-md">Copy and manage your public and private keys</p>
+                <RiKey2Line className="h-8 w-8" />
+                <h2 className="text-foreground-950 text-md font-medium">Environment Keys</h2>
+                <p className="text-foreground-400 text-xs">Copy and manage your public and private keys</p>
```

Comment:
A small suggestion: we let users choose whether to copy or not, so the explanation about copying feels unnecessary.

```suggestion
                <p className="text-foreground-400 text-xs">Manage your public and private keys</p>
```